AllReadyApp/Web-App/AllReady.UnitTest/Controllers/RequestStatusControllerTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿using AllReady.Controllers;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace AllReady.UnitTest.Controllers
+{
+    public class RequestStatusControllerTests
+    {
+        [Fact]
+        public async void NonMatchingRequestId_IndexReturnsNotFoundView() {
```

Comment:
Can you change the return type from `void` to `Task`?  That way, the test runner awaits the completion of the task before declaring success or failure.